src/config-manager.ts

```diff
@@ -13,19 +13,19 @@ interface Config {
 }
 
 export class ConfigManager {
-  private config: Config;
+  private _config: Config;
```

Comment:
nit: do we need the `_` as a prefix as a convention? I imagine since you have `private` in the TypeScript class field, it should serve as `_` no?

Reply:
We do since to avoid a naming collision as we're exposing a getter for `config`